docs/source/cli.rst

```diff
@@ -19,15 +19,18 @@ Read it
 .. code-block:: bash
 
     $ undebt --help
```

Comment:
there's a way to generate these, I forget how.  We do something like this in either dumb-init or pgctl iirc